src/pages/workspaces/migration/migration-utils.ts

```diff
@@ -73,13 +94,24 @@ export const parseServerResponse = (
       if (status === null) {
         return { namespace, name, migrationStep: 'Unscheduled' };
       }
+      const getFailureMessage = () => {
+        let failureMessage;
+        if (_.isObject(status.outcome)) {
+          try {
+            failureMessage = JSON.parse(status.outcome.failure).message;
+          } catch {
+            failureMessage = status.outcome.failure;
```

Comment:
I have encountered failure messages where the string is a JSON string, and also failures where it is just a plain string. Ie.,

```
outcome: {
          failure: 'io.grpc.StatusRuntimeException: NOT_FOUND: Service account',
        }

vs.

outcome: {
      failure:
        '{"billingProjectBillingAccount":"RawlsBillingAccountName(billingAccounts/00708C-45D19D-27AAFA)","migrationId":"3","workspace":"CARBillingTest/testdata","billingProject":"RawlsBillingProjectName(CARBillingTest)","message":"The bucket migration failed while removing workspace bucket IAM: invalid billing account on billing project."}',
    }

Reply:
At least I have unit test coverage for both cases!